files/pt-br/web/javascript/reference/global_objects/string/charcodeat/index.md

```diff
@@ -77,41 +77,6 @@ function fixedCharCodeAt(str, idx) {
 }
 ```
 
-### Corrigindo o `charCodeAt()` para manipular caracteres de Plano Multilingual não Básico se sua presença na string é desconhecida
```

Comment:
Duplicate heading, and the following content will be removed from the English document